Tools/Scripts/webkitpy/benchmark_runner/benchmark_runner.py

```diff
@@ -182,8 +181,10 @@ def _run_benchmark(self, count, web_root):
                     raise error
                 else:
                     self._browser_driver.restore_env()
-                if self._pgo_profile_output_dir:
-                    shutil.copytree(self._pgo_profile_output_dir, self._diagnose_dir, dirs_exist_ok=True)
+                if self._generate_pgo_profiles:
+                    _log.info(f'Copying PGO profiles from {self._browser_driver.pgo_profile_output_directory} '
+                              f'to {self._diagnose_dir}')
+                    self._browser_driver.collect_pgo_profile(self._diagnose_dir)
```

Comment:
It's a little strange to see this logging here, and not inside `_browser_driver.collect_pgo_profile()`. It's like we're guessing what this function will do.

Reply:
Addressed in updated patch.